mmaction/core/bbox/transforms.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+import torch
+
+
+def bbox_overlaps(bboxes1, bboxes2, mode='iou', aligned=False):
```

Comment:
Seemly it can directly be imported from `mmdet` ?

Reply:
The problem is, the bbox_overlaps in `mmdet` only works for cuda tensors. Thus I can not use it for unittest. So that I have to add this function in `mmaction2`.